lib/wallet/integration/src/Test/Integration/Scenario/API/Byron/Transactions.hs

```diff
@@ -353,18 +354,142 @@ spec = describe "BYRON_TRANSACTIONS" $ do
-                (`shouldBe` Quantity (faucetAmt + amt)) rb
+            verify rTx
+                [ expectSuccess
+                , expectField
+                        (#balance . #available)
+                        (`shouldBe` Quantity amt)
+                ]
+
+    describe "BYRON_TRANS_CREATE_01a - Single Output Transaction Byron -> Byron" $
+        forM_ [(emptyRandomWallet, "Byron wallet"), (emptyIcarusWallet, "Icarus wallet")] $
+        \(emptyByronWallet,name) -> it name $ \ctx -> runResourceT $ do
+
+        (wByron, wDestByron) <- (,) <$> fixtureRandomWallet ctx <*> emptyByronWallet ctx
+        ra <- request @ApiByronWallet ctx (Link.getWallet @'Byron wDestByron) Default Empty
+        let walType = getFromResponse #discovery ra
+
+        destination <- case walType of
+            DiscoveryRandom -> do
+                let payloadAddr = Json [json| { "passphrase": #{fixturePassphrase} }|]
+                rA <- request @(ApiAddress n) ctx (Link.postRandomAddress wDestByron) Default payloadAddr
+                pure $ getFromResponse #id rA
+            DiscoverySequential -> do
+                let link = Link.listAddresses @'Byron wDestByron
+                rA2 <- request @[ApiAddress n] ctx link Default Empty
+                let addrs = getFromResponse Prelude.id rA2
+                pure $ (addrs !! 1) ^. #id
+
+        let amt = 2 * minUTxOValue (_mainEra ctx) :: Natural
+        let payload = Json [json|{
+                "payments": [{
+                    "address": #{destination},
+                    "amount": {
+                        "quantity": #{amt},
+                        "unit": "lovelace"
+                    }
+                }]
+            }|]
+
+        rFeeEst <- request @ApiFee ctx
+            (Link.getTransactionFeeOld @'Byron wByron) Default payload
+        verify rFeeEst
+            [ expectSuccess
+            , expectResponseCode HTTP.status202
+            ]
+        let (Quantity feeEstMin) = getFromResponse #estimatedMin rFeeEst
+        let (Quantity feeEstMax) = getFromResponse #estimatedMax rFeeEst
+
+        -- make transaction from Byron wallet to Byron wallet
+        let payloadTx = Json [json|{
+                "payments": [{
+                    "address": #{destination},
+                    "amount": {
+                        "quantity": #{amt},
+                        "unit": "lovelace"
+                    }
+                }],
+                "passphrase": #{fixturePassphrase}
+            }|]
+        rTx <- request @(ApiTransaction n) ctx
+            (Link.createTransactionOld @'Byron wByron) Default payloadTx
+        verify rTx
+            [ expectSuccess
+            , expectResponseCode HTTP.status202
+            , expectField (#amount . #getQuantity) $
+                between (feeEstMin + amt, feeEstMax + amt)
+            , expectField (#direction . #getApiT) (`shouldBe` Outgoing)
+            , expectField (#fee . #getQuantity) (`shouldBe` feeEstMax)
```

Comment:
Same comments as above. I find it curious that these tests seem to pass. 🤔

Reply:
I believe they pass because of `fixtureRandomWallet` utxo distribution, which is always 10 utxos of the same size, and therefore `feeEstMax == feeEstMin`. (because it is the first tx from this wallet)